README.md

```diff
@@ -23,8 +23,9 @@ The following instructions deploy Mattermost in a production configuration using
 
 If you want to install Enterprise Edition, you can skip this section.
 
-To install the team edition, uncomment out these lines in docker-compose.yaml file:
+To install the team edition, change `build: app` to `build:` and uncomment out these lines in docker-compose.yaml file:
```

Comment:
I think the "build:" is easy to miss, it would be a good idea to just add one more line to the example below so that one can see the full change.

Reply:
essentially adding this as example:
```
  app:
    build:
      context: app
      args:
        - edition=team
```

Reply:
I made the changes as pointed out by you in `README.md` as well as the relevant part in the `docker-compose.yml` 
Review whenever you can
Thanks